core/src/main/kotlin/transformers/documentation/DefaultDocumentationNodeMerger.kt

```diff
@@ -51,15 +51,18 @@ fun Function.mergeWith(other: Function) = Function(
     if (receiver != null && other.receiver != null) receiver.mergeWith(other.receiver) else null,
     merge(parameters + other.parameters, Parameter::mergeWith),
     expected?.mergeWith(other.expected),
-    (actual + other.actual).merge()
+    (actual + other.actual).merge(),
+    visibility = visibility
```

Comment:
This can be a problem when we merge things with the same name but different visibility (same below)